com.unity.render-pipelines.high-definition/Editor/RenderPipeline/Raytracing/RayTracingSettingsEditor.cs

```diff
@@ -0,0 +1,46 @@
+using UnityEngine;
+using UnityEngine.Rendering;
+using UnityEngine.Rendering.HighDefinition;
+
+namespace UnityEditor.Rendering.HighDefinition
+{
+    [CanEditMultipleObjects]
+    [VolumeComponentEditor(typeof(RayTracingSettings))]
+    class RayTracingSettingsEditor : VolumeComponentEditor
+    {
+        SerializedDataParameter m_RayBias;
+        SerializedDataParameter m_ExtendCulling;
+        SerializedDataParameter m_ShadowCullingRange;
+
+        public override void OnEnable()
+        {
+            base.OnEnable();
+
+            var o = new PropertyFetcher<RayTracingSettings>(serializedObject);
+
+            m_RayBias = Unpack(o.Find(x => x.rayBias));
+            m_ExtendCulling = Unpack(o.Find(x => x.extendCulling));
+            m_ShadowCullingRange = Unpack(o.Find(x => x.shadowCullingRange));
+        }
+
+        public override void OnInspectorGUI()
+        {
+            HDRenderPipelineAsset currentAsset = HDRenderPipeline.currentAsset;
+            if (!currentAsset?.currentPlatformRenderPipelineSettings.supportRayTracing ?? false)
+            {
+                EditorGUILayout.Space();
+                EditorGUILayout.HelpBox("The current HDRP Asset does not support Ray Tracing.", MessageType.Error, wide: true);
+                return;
+            }
+
+            PropertyField(m_RayBias);
+            PropertyField(m_ExtendCulling);
+            if (m_ExtendCulling.overrideState.boolValue && m_ExtendCulling.value.boolValue)
```

Comment:
We decided that we should not use this pattern anymore in simple cases like that (where some parameters are hidden/disabled depending on others). Reason being that if a users just want to override a dependent parameters they can't without overriding the full chain of parameters.
For a simple case like this, just having indentation is enough to express dependency.